alica_engine/src/engine/syncmodule/SyncModule.cpp

```diff
@@ -26,25 +26,40 @@ using std::pair;
 using std::shared_ptr;
 using std::vector;
 
-SyncModule::SyncModule(const AlicaEngine* ae)
+SyncModule::SyncModule(ConfigChangeListener& configChangeListener, const TeamManager& teamManager, const PlanRepository& planRepository,
+        const IAlicaCommunication& communicator, const AlicaClock& clock)
         : _myId(0)
-        , _ae(ae)
         , _running(false)
         , _ticks(0)
+        , _configChangeListener(configChangeListener)
+        , _teamManager(teamManager)
+        , _planRepository(planRepository)
+        , _communicator(communicator)
+        , _clock(clock)
 {
+    auto reloadFunctionPtr = std::bind(&SyncModule::reload, this, std::placeholders::_1);
+    _configChangeListener.subscribe(reloadFunctionPtr);
+    reload(_configChangeListener.getConfig());
 }
+
 SyncModule::~SyncModule()
 {
     for (auto iter : _synchProcessMapping) {
         delete iter.second;
     }
 }
+
+void SyncModule::reload(const YAML::Node& config)
+{
+    _maySendMessages = !config["Alica"]["SilentStart"].as<bool>();
```

Comment:
in a later pr, we can get rid of _maySendMessages everywhere and should only have it as part of communication